lib/blogs_api/user.ex

```diff
@@ -27,13 +27,15 @@ defmodule BlogsApi.User do
     %__MODULE__{}
     |> cast(params, @allowed_params)
     |> validate_required(@required_params)
+    |> validate_required(:email, message: "\"email\" is required")
+    |> validate_required(:password, message: "\"password\" is required")
```

Comment:
Aqui você está validando o email e a senha separadamente para poder exibir a mensagem correta, certo? Então não precisa validar duas vezes.
```suggestion
    |> cast(params, @allowed_params)
    |> validate_required(:email, message: "\"email\" is required")
    |> validate_required(:password, message: "\"password\" is required")
```

Reply:
É por causa da mensagem de erro que tem que vir de forma específica. Eu tentei tirar, mas o código quebrou... :(